src/main/java/se/lth/base/server/rest/UserResource.java

```diff
@@ -137,4 +137,19 @@ public void deleteUser(@PathParam("id") int userId) {
             throw new WebApplicationException("You are not permitted to delete this user", Response.Status.FORBIDDEN);
         }
     }
+    
+    @Path("updateSessionLastSeen")
```

Comment:
This should not be its own rest call, we should add it in the authentication provider instead...

Reply:
Not sure what you refer to when you say the authentication provider.